packages/ui-voip/src/components/VoipPopupDraggable/DraggableCore.ts

```diff
@@ -0,0 +1,310 @@
+import { Emitter, OffCallbackHandler } from '@rocket.chat/emitter';
+import { useSafeRefCallback } from '@rocket.chat/ui-client';
+import { useCallback, useRef } from 'react';
+
+const DRAG_START_EVENTS = ['mousedown', 'touchstart'] as const;
+const DRAG_END_EVENTS = ['mouseup', 'touchend'] as const;
+const DRAG_MOVE_EVENTS = ['mousemove', 'touchmove'] as const;
```

Comment:
Have you considered using pointer events instead? 
They’re well supported in modern browsers and can help unify mouse and touch event handling.
Depending on how old the browsers we need to support are, this might be a good improvement.

Reply:
I am actually, @tassoevan *pointed* that out too (pun-intended)

But this was supposed to be still a draft, I don't remember making it ready, maybe I forgot to open it as a draft. Thanks for the review though.